ci-operator/config/opendatahub-io/kubeflow/opendatahub-io-kubeflow-master.yaml

```diff
@@ -47,7 +47,24 @@ tests:
       COMPUTE_NODE_TYPE: n2-standard-8
     test:
     - as: e2e
-      commands: ./opendatahub-io-kubeflow-e2e-commands.sh
+      commands: |
+        unset GOFLAGS
+        pushd components/odh-notebook-controller
+        oc new-project odh-notebook-controller-system
+        IFS=':' read -r -a CTRL_IMG <<< "${ODH_NOTEBOOK_CONTROLLER_IMAGE}"
+        export IMG="${CTRL_IMG[0]}"
+        export TAG="${CTRL_IMG[1]}"
+        export K8S_NAMESPACE="odh-notebook-controller-system"
+        cp $KUBECONFIG /tmp/kubeconfig
+        chmod 644 /tmp/kubeconfig
+        export KUBECONFIG=/tmp/kubeconfig
```

Comment:
you should try to make your test code resilient to the read-only `KUBECONFIG`, the scripts will be safer to use. This means passing `--skip-config-write` to `oc new-project` and always using `-n $namespace`.

Reply:
Agreed.  We have added that as a item on plans to update the e2e test suite